### PR TITLE
add whatsnew-76

### PR DESCRIPTION
**Pull request by Errolyn**
Bedrock PR: https://github.com/mozilla/bedrock/pull/8782
L10N repo: https://github.com/mozilla-l10n/www.mozilla.org/pull/390

URL: https://www.mozilla.org/firefox/76.0/whatsnew/all/

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -409,6 +409,14 @@
         ],
         'supported_locales' => $firefox_locales,
     ],
+    'firefox/whatsnew_76.lang' => [
+        'deadline'          => '2020-04-30',
```
Comment (peiying2) [inline app/config/sources.inc.php:413]: The deadline should be 4/24 this Friday. The completed locale list is due to releng 10 days before the launch. Please change.

Comment (peiying2) [reply app/config/sources.inc.php:413]: Could you confirm what the fallback page is if WNP76 is incomplete for non-P1 locales?

Comment (Errolyn) [reply app/config/sources.inc.php:413]: It would be our standard fallback page: https://www.mozilla.org/en-US/firefox/whatsnew/all/

**Comment by flodolo** [comment]
The build is green on Travis, but for some reason the result doesn't get back to GH.

**Pull request merged**